parcels/include/parcels.h

```diff
@@ -73,6 +73,65 @@ static inline ErrorCode spatial_interpolation_bilinear(double xsi, double eta, f
   return SUCCESS;
 }
 
+/* Bilinear interpolation routine for 2D grid for tracers with inverse distance weighting near land*/
+static inline ErrorCode spatial_interpolation_bilinear_invdist_land(double xsi, double eta, float data[2][2], float *value)
+{
+  int k, l, nb_land = 0, land[2][2] = {{0}};
+  float weight[2][2] = {{0.}}, w_sum = 0.;
+  // count the number of surrounding land points (assume land is where the value is close to zero)
+  for (int i = 0; i < 2; i++) {
+    for (int j = 0; j < 2; j++) {
+      if (fabs(data[i][j]) < 1e-14) {
```

Comment:
is this 'new' tolerance threshold required ? Also, `data` is a float, so `1e-14` is beyond the numerical resolution of `data`. How about using either 
```
if(fabs(data[i][j] < FLT_EPSILON) { ... }
```
or the pre-build function:
```
if(is_zero_flt(data[i][j])) { ... }
```
?